sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetCompressionCodecPrecedenceSuite.scala

```diff
@@ -105,7 +105,7 @@ class ParquetCompressionCodecPrecedenceSuite extends ParquetTest with SharedSpar
 
   test("Create parquet table with compression") {
     Seq(true, false).foreach { isPartitioned =>
-      Seq("UNCOMPRESSED", "SNAPPY", "GZIP").foreach { compressionCodec =>
+      Seq("UNCOMPRESSED", "SNAPPY", "GZIP", "LZ4", "ZSTD").foreach { compressionCodec =>
```

Comment:
`org.apache.hadoop.io.compress.ZStandardCodec` is added at Apache Hadoop 2.9.0+. Does the test pass with Hadoop 2 profile?

Reply:
cc @viirya too

Reply:
Parquet uses its own ZSTD codec, see https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/ZstandardCodec.java

Reply:
👌

Reply:
yea

Reply:
I have test in my own env with the "LZ4", "ZSTD" and "BROTLI". The "BROTLI" is a hadoop build in plugin, we can not check this compression, and the "ZSTD" and "LZ4" import jar packages to support those compression. So I think we need to check those compression can work well when we upgrade the parquet , zstd and lz4 version.